content-learn/mastering-meshery-istio/advance-concepts-of-service-mesh/index.mdx

```diff
@@ -31,8 +31,7 @@ meshesYouLearn:
 toc:
   [
     'getting-started',
-    'expose-services',
-   
+    'deploy-an-application',
```

Comment:
Instead of removing, you should order it correctly.

Reply:
Yes I would do it, today..